docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+output: github_document
+---
+
+```{r, echo = FALSE, warning= FALSE}
+knitr::opts_chunk$set(
+  collapse=TRUE,
+  comment = "#>",
+  fig.path = "figures/aug-"
+)
+```
+
+# Using `arcadiathemeR` to make figures in R
+
+`arcadiathemeR` is an R package that provides functions to create `ggplot2`-style figures that (mostly) adhere to the Arcadia Science style guide. This lesson provides a quick introduction to `arcadiathemeR` and how to use it to style `ggplot2`-style plots, which largely follows the [package README documentation](https://github.com/Arcadia-Science/arcadiathemeR?tab=readme-ov-file#arcadiathemer). 
+
+We'll cover the following topics:
+
+1. Installing `arcadiathemeR`
+2. Using `arcadiathemeR` to layer onto a `ggplot2` plot
+3. Accessing and using colors, palettes, and gradients
+4. Saving plots
+
+## Prerequisites and Installation
+1. You need `R` installed, and `arcadiathemeR` requires at least R version >= 4.0 (the package was built and tested with version 4.3.1). Install `R` for Mac OSx [here]
+2. You can use `RStudio` or a Jupyter notebook for creating plots. Install `RStudio` for Mac OSx [here] or Jupyter in a conda environment [following these instructions]()
+3. To use the custom fonts you need to download the `TTF` formatted font files and place in the `Users/YOURUSERNAME/Library/Fonts/` directory. You can also double click on the fonts to install them using FontBook. Check out the Arcadia Science Brand Assets page in Notion to find these. This should only need to be performed once even if the package is updated over time. **You must download the TTF formatted files to be compatible with `arcadiathemeR`.** If you want to download both the `TTF` and `OTF` formatted files to use both `arcadiathemeR` and the `arcadia-pycolor` python package that's fine - there will just be duplicate selections for each font in Illustrator. 
+4. Install `arcadiathemeR` with the `remotes` package:
+
+    ```
+    # install.packages("remotes")
+    remotes::install_github("Arcadia-Science/arcadiathemeR")
+    ```
+5. Load the package to use in your scripts with `library(arcadiathemeR)`. When you first load the package it will print a message about whether the custom fonts have been accessed and loaded correctly.
+
+## Layering onto an existing `ggplot2` plot
+
+There are two main functions you will use to layer onto an existing `ggplot2` plot to create plots adhering to the Arcadia style guide and access palettes. These are the `theme_arcadia()` and `scale` functions. The particular `scale` function differs on whether you use `color` or `fill` to access colors. Here is how you use these functions without changing the default arguments:
```

Comment:
Nit: parentheses after function names may not be necessary

```suggestion
There are two main functions you will use to layer onto an existing `ggplot2` plot to create plots adhering to the Arcadia style guide and access palettes. These are the `theme_arcadia` and `scale` functions. The particular `scale` function differs on whether you use `color` or `fill` to access colors. Here is how you use these functions without changing the default arguments:
```

Reply:
fixed all instances of this